src/main/resources/synthetic.xml

```diff
@@ -11,7 +11,6 @@
     <type type="xldeploy.Task" extends="xlrelease.PythonScript" virtual="true">
         <property name="iconLocation" default="xldeploy/xldeploy-icon-black-small.png" hidden="true" />
         <property name="taskColor" hidden="true" default="#68B749" />
-
```

Comment:
Is there a way you can put the newlines back? I'm especially concerned about the distinction between input properties and output properties.